node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,12 @@
+
+const fs = require('fs');
+
+const data = Math.floor(Math.random() * 100);
+
+const text = data.toString() + '\n';
```

Comment:
Since you are concating your number to a string you don't need the `toString`, it will already get converted to a string.